### PR TITLE
added module declaration

### PR DESCRIPTION
**Pull request by peeveen**
Suggested fix for https://github.com/moscajs/aedes/issues/878

### CODE REVIEW

**Review by robertsLando — COMMENTED** [review]
Could you add a typescript test for this? Just to check it's working as expected

**Comment by peeveen** [comment]
> Could you add a typescript test for this? Just to check it's working as expected

I spent a few hours trying this. I tried to create a whole other test "project" inside the `test` folder, with it's own `package.json` with `"type": "module"` in it, and make a relative local reference to the parent `aedes` package as a dependency, which became a circular folder symlink/shortcut ... but I think node/tap/npm/tsc/something is getting confused, because I can't get my test code to stop complaining about "cannot use namespace as type" unless I add `"type": "module"` to the __main project__ `package.json` ... which causes all manner of other problems.

Basically, I gave up.

But adding those three lines into the `node_modules/aedes/aedes.d.ts` file in my project is making everything work beautifully.

If the lack of an internal test is a big problem, just reject this. I'll keep using the workaround.

**Review by robertsLando — APPROVED** [review]
It's ok, no worries

**Comment by hjdhjd** [comment]
This PR breaks TypeScript modules with the following error:

```
node_modules/aedes/aedes.d.ts:11:3 - error TS2666: Exports and export assignments are not permitted in module augmentations.
```

In general this isn't a good idea, as I understand it. As of right now, Aedes is unusable in even the simplest of TypeScript projects.
